sdk/python/kfp/dsl/_pipeline.py

```diff
@@ -104,6 +106,21 @@ def set_ttl_seconds_after_finished(self, seconds: int):
     self.ttl_seconds_after_finished = seconds
     return self
 
+  def set_pod_disruption_budget(self, min_available: str, selector: str=None):
```

Comment:
```suggestion
  def set_pod_disruption_budget(self, min_available: str, selector: str = None):
```

Reply:
Will fix it but avoid suggestion function. See comment above.